breeze

```diff
@@ -16,146 +16,245 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-# Bash sanity settings (error on exit, complain for undefined vars, error when pipe fails)
 set -euo pipefail
 
-export BREEZE=true
-
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-export AIRFLOW_SOURCES="${MY_DIR}"
-
-# Directory where all CI scripts are located
-export SCRIPTS_CI_DIR="${MY_DIR}/scripts/ci"
-
-BUILD_CACHE_DIR="${MY_DIR}/.build"
-FILES_DIR="${MY_DIR}/files"
-TMP_DIR="${MY_DIR}/tmp"
-
-mkdir -pv "${BUILD_CACHE_DIR}"
-mkdir -pv "${TMP_DIR}"
-mkdir -pv "${FILES_DIR}"
-
-# Note - we do not use __script_init.sh here because it can only be used from within
-# the CI directory and we need to overrride PYTHON_VERSION based on what we store
-# in the .build directory
+# Bash arrays need to be defined outside of functions unfortunately :(
+# Array with extra options for Docker compose
+declare -a EXTRA_DC_OPTIONS
+# Array with selected integrations
+declare -a INTEGRATIONS
+# This is where remaining args are passed
+declare -a REMAINING_ARGS
 
-# Beginning of the initialisation here
+function setup_default_breeze_variables() {
+    # Whether to actually run docker compose with the command set given
+    export COMMAND_TO_RUN="enter_breeze"
```

Comment:
BTW you don't need to export variables form functions to make them visible outside/after the function, only if you need them to be available in subcommands. (You have to explicitly mark variables as `local var` to limit the scope)

Reply:
Yeah. I will review it and cleanup all the exports :). That's a good point

Reply:
Reviewed